op-acceptance-tests/tests/interop/mocks_test.go

```diff
@@ -29,7 +28,7 @@ var (
 	_ system.Wallet = (*mockFailingWallet)(nil)
 
 	// Ensure mockFailingChain implements Chain
-	_ system.Chain = (*mockFailingChain)(nil)
+	_ system.Chain = (*mockFailingL1Chain)(nil)
```

Comment:
Should we stick to the original name for consistency? Wdyt

Reply:
Sure makes sense.